src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -15,108 +15,148 @@
 
 error_reporting(-1);
 
-$getEnvVar = function ($name, $default = false) {
-    if (false !== $value = getenv($name)) {
-        return $value;
-    }
+global $argv, $argc, $PHP, $getEnvVar;
```

Comment:
How about passing them as contexts for functions rather than using a `global` ?

Reply:
@Taluu followed your suggestion and did avoid any usage of global